specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/examples/event-grid-schema/email_delivery_report_received.json

```diff
@@ -5,11 +5,13 @@
   "data": {
     "sender": "test2@contoso.org",
     "recipient": "test1@contoso.com",
+    "internetMessageId": "<messageid@domain.com>",
```

Comment:
All of the other example data elements include fake data so it would be better to follow that pattern. Going along with the other data elements, I would expect this to end with @contoso.com or @mx.contoso.com. We could also drop the `<` and `>` and add a fake message id.

Reply:
InternetMessageId should start with "<" and end with ">" so this part should be good. 
<xyz.abc.123@contoso.com>